### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ericsche**
Typo there was 2 time Azure Open AI vs Open AI & Azure Open AI

### CODE REVIEW

**Review by ChetanSharma-msft — APPROVED** [review]
Looks fine

File: samples/bot-ai-enterprise-search/nodejs/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ This bot has been created using [Bot Builder](https://www.npmjs.com/package/botb
 - An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/microsoft-365/dev-program)
 - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
 - [Visual Studio Code](https://code.visualstudio.com/download)
-- [Azure Open AI](https://platform.openai.com/docs/quickstart/build-your-application) or [Azure OpenAI](https://azure.microsoft.com/free/)
+- [Open AI](https://platform.openai.com/docs/quickstart/build-your-application) or [Azure OpenAI](https://azure.microsoft.com/free/)
```
Comment (ChetanSharma-msft) [inline samples/bot-ai-enterprise-search/nodejs/README.md:42]: Could you please update this URL as well: 
[Azure OpenAI](https://azure.microsoft.com/free/)

With below
https://learn.microsoft.com/en-us/azure/ai-services/openai/quickstart?tabs=command-line&pivots=programming-language-studio

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added review comment

**Review by ChetanSharma-msft — APPROVED** [review]
Looks fine, Approving the PR.

**Review by Mohammed-MSFT — APPROVED** [review]
Approving, changes looks fine.

**Pull request merged**